docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,465 @@
+| MerchantProductOptionGroupDataImportPlugin | Validates Merchant reference and inserts merchant product option groups into DB. | None | Spryker\Zed\MerchantProductOptionDataImport\Communication\Plugin\DataImport |
+| MerchantProductOptionListActionViewDataExpanderPlugin | Expands data with merchant collection. | None | Spryker\Zed\MerchantGui\Communication\Plugin\ProductOptionGui |
+| MerchantProductOptionListTableQueryCriteriaExpanderPlugin | Expands `QueryCriteriaTransfer` with merchant product option group criteria for expanding default query running in `ProductOptionListTable`. | None | Spryker\Zed\MerchantProductOptionGui\Communication\Plugin\ProductOptionGui |
+| MerchantProductOptionGroupExpanderPlugin | Expands a product option group data with related merchant. | None | Spryker\Zed\MerchantProductOption\Communication\Plugin\ProductOption |
+| MerchantProductOptionCollectionFilterPlugin | Filters merchant product option group transfers by approval status and excludes product options with not approved merchant groups. | None | Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\ProductOptionStorage |
+| MerchantProductOptionGroupWritePublisherPlugin | Retrieves all abstract product ids using merchant product option group ids from event transfers. | None | Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\Publisher\MerchantProductOption |
+
+
+**src/Pyz/Zed/Cart/CartDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Cart;
+
+use Spryker\Zed\MerchantProductOption\Communication\Plugin\Cart\MerchantProductOptionCartPreCheckPlugin;
+use Spryker\Zed\Cart\CartDependencyProvider as SprykerCartDependencyProvider;
+
+class CartDependencyProvider extends SprykerCartDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\CartExtension\Dependency\Plugin\CartPreCheckPluginInterface[]
+     */
+    protected function getCartPreCheckPlugins(Container $container): array
+    {
+        return [
+            // ...
+            new MerchantProductOptionCartPreCheckPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Checkout/CheckoutDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Checkout;
+
+use Spryker\Zed\MerchantProductOption\Communication\Plugin\Checkout\MerchantProductOptionCheckoutPreConditionPlugin;
+use Spryker\Zed\Checkout\CheckoutDependencyProvider as SprykerCheckoutDependencyProvider;
+
+class CheckoutDependencyProvider extends SprykerCheckoutDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\CheckoutExtension\Dependency\Plugin\CheckoutPreConditionPluginInterface[]
+     */
+    protected function getCheckoutPreConditions(Container $container)
+    {
+        return [
+            // ...
+            new MerchantProductOptionCheckoutPreConditionPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/DataImport/DataImportDependencyProvider.php**
```

Comment:
Should be part of `Import Data` section. See  https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides for the allowed sections list.